components/connector/go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/kyma-incubator/compass/components/connector
+
+go 1.15
+
+require (
+	cloud.google.com/go v0.44.3 // indirect
+	github.com/99designs/gqlgen v0.9.3
+	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/google/uuid v1.1.2 // indirect
+	github.com/googleapis/gnostic v0.3.1 // indirect
+	github.com/gorilla/mux v1.7.3
+	github.com/kyma-incubator/compass/components/director v0.0.0-20201215200050-802603cbf300
```

Comment:
Why does the Director reference have to be of a specific commit? (maybe same question for the machinebox/grapql)

Reply:
It seems that `k8s.io/kube-openapi` is also in the same fashion. I'm also wondering.

Reply:
Versions are supported only for repositories that tag their commits accordingly. We currently don't do that in compass so that is why it looks like that for director. On the other side machinebox/graphql have tagged only a couple of versions (in 2018) and have stopped tagging ever since. The commit + hash that we've got means _"the most recent versioned commit before the target commit is vX.Y.Z."_  
Reference: [Go Pseudo versions](https://golang.org/cmd/go/#hdr-Pseudo_versions); [machinebox/graphql tags](https://github.com/machinebox/graphql/tags)
On the other hand, k8s.io probably do tag their commits properly but since that is an indirect dependency I would suggest not changing it rn.

Reply:
We don't have versions in compass. If you reference a branch go mod stores the last commit reference of that branch I think.

Reply:
Got it - thanks for the clarification guys! 😊

P.S. @krasish I'd also suggest not changing the k8s.io dependency right now.